src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,33 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, setTubId, tabId }) => {
```

Comment:
<img width="473" alt="image" src="https://github.com/mate-academy/react_tabs-js/assets/70220667/b7f5ef27-79fc-4300-a529-0f490122b683">
please, rename the props according to the task description